website/content/docs/ecs/enterprise.mdx

```diff
@@ -59,7 +56,68 @@ features.
 | Advanced Federation/Network Areas | Yes (servers) | This feature runs on Consul servers.                                                                                                                                    |
 | Sentinel                          | Yes (servers) | This feature runs on Consul servers.                                                                                                                                    |
 | Network Segments                  | No            | Currently there is no capability to configure the network segment Consul clients on ECS run in.                                                                         |
-| Namespaces                        | No            | Currently there is no capability to configure the Consul namespace for a service on ECS.                                                                                |
-| Admin Partitions                  | No*           | Supported if Consul ECS clients run in the default partition. Otherwise there is currently no capability to configure the admin partition Consul clients in ECS run in. |
+| Namespaces                        | Yes           | This feature requires Consul Enterprise servers and clients.                                                                                                            |
```

Comment:
nit: One exception is OSS clients can register into the default namespace and partition. So it might be good to add that caveat, maybe just as a note somewhere?

Reply:
Good call. I have added a note to each row.